client/src/components/App.js

```diff
@@ -33,38 +31,34 @@ class App extends Component {
   async showProjects() {
     const response = await fetch('/projects');
     let projects = await response.json();
-    let new_projects = projects.filter(project => project.status === 'new');
-    let in_progress_projects = projects.filter(project => project.status === 'in-progress');
-    let done_projects = projects.filter(project => project.status === 'done');
 
-    this.setState({new_projects, in_progress_projects, done_projects});
+    this.setState({projects});
   }
 
   render() {
+    let project_collections = {};
```

Comment:
What about using an actual Map for this, so that we don't need to do all the dumb stuff with `hasOwnProperty`?

Reply:
The `hasOwnProperty` stuff was to please the linter. I'll see what it thinks of map.